modules/microshift-restoring-backups.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-restoring-backups_{context}"]
+= Restoring backups using the auto-recovery feature
+
+You can restore backups after system events that remove or damage required data.
+
+You can use the following procedure to restore backups.
+
+.Prerequisites
+
+* You have stopped {microshift-short}.
+
+.Procedure
+
+. Restore the backup from the directory in which you have stored the backups by running the following command:
++
+[source,terminal]
+----
+$ sudo microshift restore --auto-recovery _<path_of_directory>_ <1>
+----
+<1> Replace `_<path_of_directory>_` with the path of the directory that stores backups. For example, `/var/lib/microshift-auto-recovery`.
++
+[NOTE]
+====
+The `--auto-recovery` option copies the {microshift-short} data to `/var/lib/microshift-auto-recovery/failed/` for later investigation, selects the most recent backup, and restores it.
+The `--dont-save-failed` option disables the backing up of failed {microshift-short} data.
+====
++
+.Example output
++
+[source,terminal]
+----
+$ sudo microshift restore --auto-recovery /var/lib/microshift-auto-recovery
```

Comment:
Same comment about removing the actual command from example output:
```suggestion
```

Reply:
done